forms-shared/tests/definitions/__snapshots__/formDefinitions.ts.snap

```diff
@@ -24816,7 +24816,9 @@ exports[`Form definitions > oznamenie-o-poplatkovej-povinnosti-za-komunalne-odpa
                     "type": "object",
                   },
                 },
-                "required": [],
+                "required": [
```

Comment:
Only those that had children required but haven't had required on themselves are updated = correct.